contracts/lib/Memory.sol

```diff
@@ -32,7 +32,7 @@ library Memory {
         return equals(addr, addr2, len);
     }
 
-    function compareStrings(string memory a, string memory b) internal pure returns (bool) {
+    function compareStrings(string memory a, string memory b) internal pure returns (bool) { //@openissue
```

Comment:
probably more efficient to start with length compare:
return a.length == b.length && keccak...'

Reply:
True, but don't think it warrants changing for now